src/main/java/core/basesyntax/controller/ConsoleHandler.java

```diff
@@ -0,0 +1,48 @@
+package core.basesyntax.controller;
+
+import core.basesyntax.dao.BetDao;
+import core.basesyntax.dao.BetDaoImpl;
+import core.basesyntax.dao.UserDao;
+import core.basesyntax.dao.UserDaoImpl;
+import core.basesyntax.model.Bet;
+import core.basesyntax.model.User;
+import java.util.Scanner;
+
+public class ConsoleHandler {
+    BetDao betDao = new BetDaoImpl();
+    UserDao userDao = new UserDaoImpl();
+
+    public void handle() {
+        Scanner scanner = new Scanner(System.in);
+
+        while (true) {
+            String input = scanner.nextLine();
+            if (input.equalsIgnoreCase("q")) {
+                return;
+            }
+            User user = null;
+            Bet bet = null;
+
+            try {
+                String[] betAndUserData = input.split(" ");
+                String userName = betAndUserData[0];
+                int userAge = Integer.parseInt(betAndUserData[1]);
+                if (userAge < 18) {
+                    throw new IllegalArgumentException();
+                }
+                user = new User(userName, userAge);
+                int value = Integer.parseInt(betAndUserData[2]);
+                double risk = Double.parseDouble(betAndUserData[3]);
+                bet = new Bet(value, risk);
+            } catch (NumberFormatException e) {
+                System.out.println("Input data is incorrect");
+            } catch (IllegalArgumentException x) {
+                System.out.println("You are too young to place bets, go back to school!");
+            }
```

Comment:
let's just print this message and move on to the next iteration instead of throwing this exception

Reply:
okay, done